src/main/java/org/reduct/client/ReductTokenClient.java

```diff
@@ -88,11 +114,14 @@ private String serializeCreateTokenBody(TokenPermissions permissions) {
       }
    }
 
-   private AccessToken parseAccessToken(String body) {
+   HttpResponse<String> sendRequest(HttpRequest getRequest) {
       try {
-         return objectMapper.readValue(body, AccessToken.class);
-      } catch (JsonProcessingException e) {
-         throw new ReductSDKException("The server returned a malformed response.", e);
+         return httpClient.send(getRequest, HttpResponse.BodyHandlers.ofString());
+      } catch (IOException e) {
```

Comment:
Why don't you use  RecuctClient.sendRequest?

Reply:
This method used by createToken, I will rework it with using ReductClient in next task. I did't change it to avoid test falling.